### PR TITLE
Use ES6 syntax.

### PR DESCRIPTION
**Pull request by mattcollier**
@dlongley @davidlehn This commit is entirely automated.  Need to determine how we're going to handle services.

### CODE REVIEW

**Comment by dlongley** [comment]
I would think services would be:

```js
/* @ngInject */
export default function factory($foo, $bar) {
  const service = {};

  // ...

  return service;
};
```

And in `main.js`:

```js
import FooService from './foo-service';

module.service('brFooService', FooService);
```

**Comment by mattcollier** [comment]
@dlongley must this be a named function `export default function factory()` or could it be an anonymous function: `export default function(...)`

**Comment by dlongley** [comment]
Having it be a named function helps with debugging, but if it needs to be anonymous for some reason, that's fine.

File: agreement-component.js [hunk 1]
```diff
@@ -2,22 +2,17 @@
  * Copyright (c) 2016-2017 Digital Bazaar, Inc. All rights reserved.
  */
 /* global requirejs */
```
Comment (davidlehn) [inline agreement-component.js]: This isn't needed anymore.  Easy enough to deal with all these by hand later.

**Comment by davidlehn** [comment]
I have no serious preference on having default function names.  If we're not using naming, could even go with fat arrow syntax: `export default (...) => {...}`.

**Review by dlongley — APPROVED** [review]

File: agreement-service.js [hunk 2]
```diff
@@ -78,7 +71,3 @@ function factory($http, config) {
 
   return service;
 }
```
Comment (davidlehn) [inline agreement-service.js:73]: Our style checkers might be unhappy without a ";" here.  There is one when exporting objects.

Comment (mattcollier) [reply agreement-service.js:73]: For the record: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

**Pull request merged**